frontend/src/modules/report/FeatureReportContainer/ScenarioList/ScenarioStep/PopperMenu.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC, MouseEvent, ReactNode, useState, useRef } from 'react';
+import { IconButton, Popper, Fade, Card, List, ListItem } from '@material-ui/core';
+import { MoreHoriz } from '@material-ui/icons';
+import { makeStyles, createStyles } from '@material-ui/core/styles';
+import { useTranslation } from 'react-i18next';
+
+import Status from 'models/Status';
+
+const { Passed, Failed, Undefined, Skipped } = Status;
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    scenarioStepIcon: {
+      fontSize: '16px',
+    },
+    moreButton: {
+      padding: '0px',
+    },
+    popperMenu: {
+      zIndex: 999,
+    },
+  })
+);
+
+interface Props {
+  stepId: string;
+  status: Status;
+  onStepStatusChange(e: MouseEvent<HTMLElement>, stepId: string, status: Status, newStatus: Status): void;
+}
+
+const PopperMenu: FC<Props> = ({ stepId, status, onStepStatusChange }) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const ref = useRef(null);
+  const [open, setOpen] = useState(false);
+
+  const labelMap = {
+    passed: t('report.pass'),
+    failed: t('report.fail'),
+    skipped: t('report.skip'),
+    undefined: t('report.undefined'),
```

Comment:
Is the (inferred) type of this map a Record<string, string> or a Record<Status, string>? The latter would be better, but it kinda looks like it might be the former?

Reply:
At a string, I forgot to put the type statement on it. Will do